js/join.js

```diff
@@ -39,6 +42,19 @@ const flowState = {
 };
 
 const startBtn = document.getElementById('start');
+const buttonContainer = document.querySelector('.button-container');
+const disclaimerCheckbox = document.getElementById('disclaimer-checkbox');
+const disclaimerFeatureFlag = document.querySelector('.disclaimer_feat-flag');
+
+window.addEventListener('load', (event) => {
+  if (queryString.includes('dev=true')) {
+    buttonContainer.classList.add('button-container_disclaimer');
+    disclaimerFeatureFlag.classList.remove('hidden');
+    startBtn.classList.remove('button-filled');
+    startBtn.classList.add('button-disabled');
```

Comment:
You're adding the class 'button-disabled' here. May I know why there is a need to again add a `startBtn.disabled = true`

Reply:
Both have different terms :

Class `button-disabled` is used to give CSS styling when a button is disabled

`startBtn.disabled = true` is an attribute that is used to disable the button, if it is true then the user can not click on this button